x-pack/plugin/core/src/test/java/org/elasticsearch/protocol/xpack/watcher/PutWatchResponseTests.java

```diff
@@ -25,6 +26,16 @@ protected PutWatchResponse doParseInstance(XContentParser parser) throws IOExcep
         return PutWatchResponse.fromXContent(parser);
     }
 
+    @Override
+    public org.elasticsearch.client.watcher.PutWatchResponse doHLRCParseInstance(XContentParser parser) throws IOException {
```

Comment:
I like `doHlrcParseInstance` or something like that. Not sure.

Reply:
done in b5372c4